src/main/java/com/aws/greengrass/clientdevices/auth/iot/registry/RegistryConfig.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.clientdevices.auth.iot.registry;
+
+public class RegistryConfig {
+    public static final int REGISTRY_SIZE = 50;
```

Comment:
Do you want this yet? You're using it as a cache size, which is different than registry size.

Reply:
Right, we want this to set the cache size. Let me rename this to `REGISTRY_CACHE_SIZE` for now.